.travis.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     - stage: test
       go_import_path: github.com/golang/dep
       install:
-        - go get -u honnef.co/go/tools/cmd/{gosimple,staticcheck}
+        - go get -u honnef.co/go/tools/cmd/{gosimple,staticcheck,unused}
```

Comment:
It looks like running them together as [megacheck](https://github.com/dominikh/go-tools/tree/master/cmd/megacheck) could be faster.

Reply:
Done.